umeboshi/models.py

```diff
@@ -112,11 +112,11 @@ def marshal_data(data):
         """
         Events use `pickle` to marshal their argument data for storage.
         """
-        return pickle.dumps(data)
+        return serializer.dumps(data)
```

Comment:
Does this imply we will be doing class loading at runtime? If so, this seems risky if we don't validate these classes exist at application initialization time...

Reply:
Validation is done in `load_class` method in `serializer.py`. If serializer user provided does not exist, the error will be raised during app initialization time.